avocado/utils/software_manager.py

```diff
@@ -409,6 +411,7 @@ def __init__(self, cmd='yum'):
         cmd_result = process.run(y_cmd, ignore_status=True,
                                  verbose=False, shell=True)
         out = cmd_result.stdout.strip()
+        out = astring.to_text(out, locale.getpreferredencoding())
```

Comment:
The same actually here although looking at this I think we could use `astring.to_text(..., errors=...)` argument. Here it'd fit perfectly as `replace` would be a good candidate in case of some corruption. What do you think @clebergnu?